api/user.go

```diff
@@ -79,30 +81,39 @@ func (a *API) UserUpdate(w http.ResponseWriter, r *http.Request) error {
 
 	err = a.db.Transaction(func(tx *storage.Connection) error {
 		var terr error
-		if params.Password != nil {
-			if len(*params.Password) < config.PasswordMinLength {
-				return invalidPasswordLengthError(config)
-			}
+		p := r.URL.Path
+		var shouldSendEmail bool
+		switch {
+			case p == "/user":	
+				if params.Password != nil {
+					if len(*params.Password) < config.PasswordMinLength {
+						return invalidPasswordLengthError(config)
+					}
+
+					if terr = user.UpdatePassword(tx, *params.Password); terr != nil {
+						return internalServerError("Error during password storage").WithInternalError(terr)
+					}
+				}
 
-			if terr = user.UpdatePassword(tx, *params.Password); terr != nil {
-				return internalServerError("Error during password storage").WithInternalError(terr)
-			}
-		}
+				if params.Data != nil {
+					if terr = user.UpdateUserMetaData(tx, params.Data); terr != nil {
+						return internalServerError("Error updating user").WithInternalError(terr)
+					}
+				}
 
-		if params.Data != nil {
-			if terr = user.UpdateUserMetaData(tx, params.Data); terr != nil {
-				return internalServerError("Error updating user").WithInternalError(terr)
-			}
-		}
+				if params.AppData != nil {
+					if !a.isAdmin(ctx, user, config.JWT.Aud) {
+						return unauthorizedError("Updating app_metadata requires admin privileges")
+					}
 
-		if params.AppData != nil {
-			if !a.isAdmin(ctx, user, config.JWT.Aud) {
-				return unauthorizedError("Updating app_metadata requires admin privileges")
-			}
+					if terr = user.UpdateAppMetaData(tx, params.AppData); terr != nil {
+						return internalServerError("Error updating user").WithInternalError(terr)
+					}
+				}
+				shouldSendEmail = true
+			case p == "/user/email":	
+				shouldSendEmail = params.ShouldSendEmail != "false"
```

Comment:
you won't have to do this if the param is of type bool, you can just use the value of `params.ShouldSendEmail` directly